opcua/gopcua/subscribe.go

```diff
@@ -101,7 +101,7 @@ func (c client) Subscribe(cfg opcua.Config) error {
 	defer oc.Close()
 
 	sub, err := oc.Subscribe(&opcuaGopcua.SubscriptionParameters{
-		Interval: 2000 * time.Millisecond,
+		Interval: 1000 * time.Millisecond,
```

Comment:
Does it make sense to use an env variable from outside of the image?

Reply:
In any case, move constants to the top. But yes, this should probably be configurable via ENV var.